MVP Module/Source/Feed UI/Controllers/FeedViewController.swift

```diff
@@ -33,6 +35,14 @@ public final class FeedViewController: UITableViewController, UITableViewDataSou
 		}
 	}
 
+	func display(_ viewModel: FeedErrorViewModel) {
+		guard let errorMessage = viewModel.errorMessage else {
+			errorView?.hideMessage()
+			return
+		}
+		errorView?.show(message: errorMessage)
```

Comment:
Suggestion: 

An `if/else` statement is usually easier to read/follow in such cases:

"If there's a message to show, show it. Else, hide the existing one."

```suggestion
		if let errorMessage = viewModel.errorMessage {
			errorView?.show(message: errorMessage)
		} else {
			errorView?.hideMessage()
		}
```

Instead of "guard against not having a message. Else, hide the existing one. Show the message if it passed the guard statement."